_notebooks/2020-07-28-lineage.ipynb

```diff
@@ -0,0 +1,323 @@
+{
+ "cells": [
+   {
+    "cell_type": "markdown",
+    "metadata": {},
+    "source": [
+     "# Record metadata on Kubeflow from Notebooks\n",
+     "> Demonstration of how lineage tracking works\n",
+     "\n",
+     "- toc: true \n",
+     "- badges: true\n",
+     "- comments: true\n",
+     "- categories: [jupyter]"
+    ]
+   },
+   {
+    "cell_type": "markdown",
+    "metadata": {},
+    "source": [
+     "# Lineage Tracking\n",
+     "* The intent of this blog is to highlight how people can use metadata to do lineage tracking.\n",
+     "* We are going to create notebook that uses the metadata SDK to write a sequence of executions and artifacts to TFX metadata.\n",
+     "* We are going to open up lineage view in the KFP UI to view the lineage\n",
+     "* Here shows explanation of functionality of metadata sdk, but if you want to try and deploy model and want to see lineage tracking UI then skip to plot lineage UI section [Plot the lineage graph](http://0.0.0.0:4000/jupyter/2020/07/28/lineage.html#Plot-the-lineage-graph)"
```

Comment:
I would write this as:

"This blog post will first guide you through the metadata SDK API, to create and log several actions to the metadata DB. Afterwards, you will be able to navigate to the Kubeflow UI and the resulting lineage graph, which gives you a graphical representation of the dependencies between the objects you logged using the SDK"

Or something similar